app/javascript/packs/src/settings/components/team/components/UpdateTeamDescriptionModal.jsx

```diff
@@ -0,0 +1,133 @@
+import React, { Component } from "react";
+import PropTypes, { bool, number, string, func } from "prop-types";
+import {
+  Modal,
+  Button,
+  FormGroup,
+  ControlLabel,
+  FormControl,
+  HelpBlock
+} from "react-bootstrap";
+import { FormattedMessage } from "react-intl";
+import _ from "lodash";
+import styled from "styled-components";
+import axios from "../../../../../app/axios";
+
+import { TEXT_MAX_LENGTH } from "../../../../../app/constants/numeric";
+import { TEAM_UPDATE_PATH } from "../../../../../app/routes";
+import { COLOR_APPLE_BLOSSOM } from "../../../../../app/constants/colors";
+
+const StyledHelpBlock = styled(HelpBlock)`
+  color: ${COLOR_APPLE_BLOSSOM}
+`;
+
+class UpdateTeamDescriptionModal extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { errorMessage: "", description: "" };
+    this.onCloseModal = this.onCloseModal.bind(this);
+    this.updateDescription = this.updateDescription.bind(this);
+    this.handleDescription = this.handleDescription.bind(this);
+    this.getValidationState = this.getValidationState.bind(this);
+  }
+
+  onCloseModal() {
+    this.setState({ errorMessage: "", description: "" });
+    this.props.hideModal();
+  }
+
+  getValidationState() {
+    if (this.state.errorMessage.length > 0) {
```

Comment:
this can be written in one line (ternary operator)

Reply:
good one :+1: